### ISSUE TITLE
1.16.1 Forge Server Mineshafts generates underwater

### ISSUE THREAD

**Issue by allethargic** [yungnickyoung/YUNGs-Better-Mineshafts#12]
Mineshafts sometimes generates underwater and it would be fine if they wasn't filled with air.

**Comment by yungnickyoung** [comment]
Please provide mod list as well as world seed and coordinates where this happens

**Comment by allethargic** [comment]
Sadly, I can't as I deleted this server and created new one with Fabric. 

But I remember few details if this can help. The only mod which changes World Generation was Biomes'o'Plenty and the biome was Warm Ocean. Generated shaft was very short and single one.

**Comment by ColdFang** [comment]
Hey, I noticed the same problem. It happened with and without BOP. With BOP installed I saw it more often but thats just a feeling. Also I think with BOP installed the mineshafts are sticking out more. I didnt test it on 1.16.2 or 3 - still waiting for some other mods in my modpack to update :) 

Mc: 1.16.1
Forge: 32.0.108
Better Mineshafts: 1.16.1-1.3
Seed: test
Coords: x:790/z:190
with BOP installed on same seed at: x-170/z:70


![2020-09-16_23 30 55](https://user-images.githubusercontent.com/13708113/93394707-ecbe2300-f874-11ea-8fa6-1659387cc66f.png)
